src/utils/index.ts

```diff
@@ -36,13 +36,20 @@ for (const [name, validationSchemas] of Object.entries(eventsValidationSchemas))
 
 export type StandardLog<T extends GenericEvent = GenericEvent> = (data: T) => (message: string) => string;
 
+function isNullish(value: string): string {
+  return value ?? "none";
+}
```

Comment:
Imo y'a moyen d'améliorer le nommage de cette fonction, le fait qu'elle commence par `is` déjà ça laisse penser que ça retourne un `boolean` mais c'est pas le cas, t'en penses quoi?
```suggestion
function logValueFallback(value: string): string {
  return value ?? "none";
}
```
Quelque chose de ce genre peut-être ?